Gemfile

```
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '~> 2.7.0'
+ruby '>= 2.7.0'
```

```suggestion
ruby '~> 2.7.0'
```
@MarienL1995 we want to use ~> because versions higher than 2.7.x are not backwards compatible ~> guarantees backwards compatibility.